src/clm5/biogeophys/BalanceCheckMod.F90

```diff
@@ -707,8 +708,10 @@ subroutine BalanceCheck( bounds, &
           ! TODO: Balance errors must be fixed for fully coupled model (ICON-eCLM-ParFlow)
           write(iulog,*)'Ignoring soil balance error...'
 #else
-          write(iulog,*)'clm model is stopping - error is greater than 1e-5 (mm)'
-          call endrun(decomp_index=indexc, clmlevel=namec, msg=errmsg(sourcefile, __LINE__))
+          if (abs(errsoi_col(indexc)) > 1.e-4_r8 .and. (DAnstep > skip_steps) ) then
```

Comment:
Good catch @jjokella ! This is an [unfortunate bug](https://github.com/HPSCTerrSys/eCLM/commit/be013178bb6f55e738f85d4144654ab4f7b85c04#diff-f305c7ba1e2a9600382ed63d80be08395df5e3c85b347420bd670a56372dc3c6) that I introduced in the [eCLM-ParFlow coupling PR](https://github.com/HPSCTerrSys/eCLM/pull/17).